src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1023,10 +1023,11 @@ bool IHostProvidesTelemetryData.HostIsInteractive
         double IHostProvidesTelemetryData.ReadyForInputTimeInMS { get { return _readyForInputTimeInMS; } }
 
         int IHostProvidesTelemetryData.InteractiveCommandCount { get { return _interactiveCommandCount; } }
+
+        private double _readyForInputTimeInMS;
 #endif
 
         private double _profileLoadTimeInMS;
-        private double _readyForInputTimeInMS;
         private int _interactiveCommandCount;
```

Comment:
It seems the field is not used too.

Reply:
Yes, looks like it's also for legacy telemetry.  Will fix.